src/management/organizers/edit/organizer-edit.html

```diff
@@ -9,7 +9,7 @@ <h1 class="title" ng-bind="oec.organizer.name"></h1>
         <div class="row">
             <div class="col-sm-12 col-md-8">
                 <div class="form-group has-feedback"
-                     ng-class="{'has-warning' : organizersWebsiteFound || organizerForm.website.$error.required }">
+                     ng-class="{'has-warning' : oec.organizersWebsiteFound || oec.organizerEditForm.website.$error.required }">
```

Comment:
In de controller gebruik je de naam '_controller_', terwijl je hier '_eoc_' in de view gebruikt. Structureel is dit natuurlijk niet fout, maar het kan verwarrend zijn.